docs/package_info/contributing.rst

```diff
@@ -6,7 +6,7 @@ an established consistent way of doing things. A consistent style increases
 readability, decreases bug-potential and makes it faster to understand how
 everything works together.
 
-PRAW follows :PEP:`8` and :PEP:`257`. You can use ``pre-push.py`` to test for
+PRAW follows :PEP:`8` and :PEP:`257`. You can use ``pre_push.py`` to test for
```

Comment:
I actually wasn't clear if this was just a typo or some outdated advice, as the current pre_push.py generates a whole lot of stuff besides a flake8 check.